File: TDDMicroExercises/Javascript/turn-ticket-dispenser/ticket-dispenser.js
TicketDispenser = function() {
};

TicketDispenser.prototype = {
	
	getTurnTicket: function () {
		
			var newTurnNumber = TurnNumberSequence.getNextTurnNumber();
			var newTurnTicket = new TurnTicket(newTurnNumber);

			return newTurnTicket;
	}

};

File: TDDMicroExercises/Java/TurnTicketDispenser/src/main/java/tddmicroexercises/turnticketdispenser/TicketDispenser.java
package tddmicroexercises.turnticketdispenser;

public class TicketDispenser
{
    public TurnTicket getTurnTicket()
    {
        int newTurnNumber = TurnNumberSequence.getNextTurnNumber();
        TurnTicket newTurnTicket = new TurnTicket(newTurnNumber);

        return newTurnTicket;
    }
}
